src/Nest/Mapping/Types/Core/String/StringAttribute.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Nest
 {
-	[Obsolete("Only valid for indices created before Elasticsearch 5.0 and will be removed in the next major version.  For newly created indices, use Text or Keyword attribute instead.")]
+	[Obsolete("Only valid for indices created before Elasticsearch 5.0, removed in NEST 6.x.  For newly created indices, use Text or Keyword attribute instead.")]
```

Comment:
Maybe `removed in Elasticsearch 6.0`? It's clear that the feature is gone from Elasticsearch and the client reflects this?

Reply:
:+1: